src/docker/02-preload/preload.sh

```diff
@@ -9,7 +9,8 @@ should_load() {
     if [[ ${file} =~ \.done$ ]]; then echo false
     elif [ -f ${file} ]; then
         if [ ! -e ${file}.done ]; then echo true
-        elif [[ $(stat -c %Y ${file}) > $(stat -c %Y ${file}.done) ]]; then echo true
+        # Removing updated behavior until `stat` returns
+        #elif [[ $(stat -c %Y ${file}) > $(stat -c %Y ${file}.done) ]]; then echo true
```

Comment:
Not too sure where stat went, but..... This can be easily changed to `elif [[ ${file} -nt ${file}.done ]]; then echo true` and then we don't loose this functionality.  btw, thanks for troubleshooting this!